packages/espanso-dropdown-list-template/espanso-dropdown-list-example-usage.yml

```diff
@@ -0,0 +1,16 @@
+matches:
+
+  - triggers: ['$colors$']
```

Comment:
Why `triggers: ` followed by a sequence, rather than a simple `trigger: `?

Reply:
I usually use triggers: ['option1', 'option2'] to give me more options when typing triggers but I edited the package trigger to be just trigger for ease of use